testsuite/ada_projects/checks/src/anonymous_subtypes.ads

```diff
@@ -0,0 +1,14 @@
+package Anonymous_Subtypes is
+
+   type T is range 0 .. 10; -- NO FLAG
```

Comment:
Running your code above on this file, this line is flagged, so the code of your checker looks wrong.